cg/models/taxprofiler/taxprofiler.py

```diff
@@ -68,13 +70,13 @@ def reformat_sample_content(self) -> list[list[str]]:
         return [
             [
                 self.name,
-                self.run_accession,
+                run_accession + 1,
```

Comment:
Should we also remove `run_accession` from the `TaxprofilerSampleSheetEntry` model?

Reply:
Removed it, resolving this.

Reply:
Has this been addressed? Did you forget to push the commit? :)

Reply:
(https://github.com/Clinical-Genomics/cg/blob/de25f2afa93234d9bf8c9ccf7dc4026f981fb684/cg/models/taxprofiler/taxprofiler.py#L75)

Reply:
cookie stuff, sorry! let's merge it